src/components/composite/poster.js

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import SbEditable from "storyblok-react";
+import { FlexBox, Heading } from "decanter-react";
+import { dcnb } from "cnbuilder";
+import CardImage from "../media/cardImage";
+import CreateBloks from "../../utilities/createBloks";
+import getNumBloks from "../../utilities/getNumBloks";
+import RichTextRenderer from "../../utilities/richTextRenderer";
+import CircularImage from "../media/circularImage";
+
+const Poster = ({
+  blok: {
+    cta,
+    borderColor,
+    isBigHeadline,
+    image: { filename },
```

Comment:
```suggestion
    image: { filename } = {},
```
I've found that add this last bit really help with preventing undefined object error (when the component is first added with no image). I've been adding it to all components that have images, but forgot the basic card - if you could help me add this bit to the basic card image, that'd be awesome 🙏🏼